kubernetes/manifest-templates/velum.yaml

```diff
@@ -82,3 +90,6 @@ spec:
     - name: velum-source-code
       hostPath:
         path: ${project_dir}
+    - name: velum-secrets
+      hostPath:
+        path: /tmp/velum-secrets
```

Comment:
I now see, we only set this to `/tmp` on dev and test. It shouldn't be a problem then.